mne/utils/docs.py

```diff
@@ -3495,6 +3553,50 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 For testing the lower tail (``tail=-1``), don't subtract ``pval`` from 1.
 """
 
+docdict['time_bandwidth_tfr'] = """
+time_bandwidth : float ``≥ 2.0``
+    Product between the temporal window length (in seconds) and the *full*
+    frequency bandwidth (in Hz). This product can be seen as the surface of the
+    window on the time/frequency plane and controls the frequency bandwidth
+    (thus the frequency resolution) and the number of good tapers. See notes
+    for additional information."""
+
+docdict['time_bandwidth_tfr_notes'] = r"""
+In MNE-Python's multitaper functions, the time-frequency trade-off is
+additionally affected by the parameter ``time_bandwidth``.
+The ``n_cycles`` parameter determines the temporal window length based on the
+frequencies of interest: :math:`T = \frac{\mathtt{n\_cycles}}{\mathtt{freqs}}`.
+The ``time_bandwidth`` parameter defines the "time-bandwidth product", which is
+the product of the temporal window length (in seconds) and the frequency
+bandwidth (in Hz). Thus once ``n_cycles`` has been set, frequency bandwidth is
+determined by :math:`\frac{\mathrm{time bandwidth}}{\mathrm{time window}}`, and
```

Comment:
@drammock The spaces don't render in this one, do you know how to fix it? Should we escape the spaces?

![Screenshot 2022-11-01 at 17 14 10](https://user-images.githubusercontent.com/73893616/199282288-63d82cf9-733a-4596-b6d2-c67f2a7990df.png)

Reply:
oops, should be `time~bandwidth` and `time~window`